lib/routes/home/widgets/payments_list/payment_item_title.dart

```diff
@@ -22,8 +22,35 @@ class PaymentItemTitle extends StatelessWidget {
   }
 
   String _title(BuildContext context) {
-    final description = _paymentInfo.description;
-    if (description != null && description.isNotEmpty) return description;   
+    final description = _paymentInfo.description?.replaceAll("\n", " ").trim();
+    if (description != null && description.isNotEmpty) {
+      final breezPosRegex = RegExp(r'(?<=\|)(.*)(?=\|)');
+      if (breezPosRegex.hasMatch(description)) {
+        final extracted = breezPosRegex.stringMatch(description)?.trim();
+        if (extracted != null && extracted.isNotEmpty) {
+          return extracted;
+        }
+      }
+      return description;
+    }
+
+    final details = _paymentInfo.details.data;
+    if (details is LnPaymentDetails) {
+      final label = details.label;
```

Comment:
We don't need to take into consideration the label as we just populate there a random value since greenlight require it to be unique: https://github.com/breez/breez-sdk/blob/main/libs/sdk-core/src/greenlight.rs#L316

Reply:
I see, thanks @roeierez label removed